docs/source/user_guide/timeseries.ipynb

```diff
@@ -253,7 +253,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The pipeline found by AutoMLSearch has a 268% improvement over the naive forecast!"
+    "The pipeline found by AutoMLSearch has a 31% improvement over the naive forecast!"
```

Comment:
Personally, I'm hesitant on putting numeric values for these since if we change/add components or change the way things are calculated, these numbers will likely change. Not blocking, but something to keep in mind!